src/pages/ProductList/ProductList.js

```diff
@@ -1,7 +1,109 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import ProductListDetails from './ProductListDetails';
+import './ProductList.scss';
 
 const ProductList = () => {
-  return <div>ProductList</div>;
+  // const [List, setList] = useState([]);
+
+  // useEffect(() => {
+  //   fetch('http://localhost:3000/data/listData.json', {
```

Comment:
목데이터를 사용하는 경우, 첫 번째 인자에서 `http://localhost:3000`은 생략해도 됩니다!
포트 번호가 3001, 3002 처럼 바뀔 경우 에러가 발생하고 일일이 수정해야 하기 때문에, 생략해주시는 게 좋습니다!
```suggestion
  //   fetch('/data/listData.json', {
```

Reply:
수정완료했습니다 :)